vcpkg/consume/caching-workflow-identity-federation.md

```diff
@@ -0,0 +1,185 @@
+> * [Set up Workload Identity Federation between Azure Storage and Azure DevOps](#2---set-up-workload-identity-federation-with-azure-devops)
+> * [Assign Managed Identity Permissions to the Storage Account Container](#3---assign-managed-identity-permissions-to-the-storage-account-container)
+> * [Generate a SAS token, and use it with Caching](#4---generate-a-sas-token-and-use-it-with-caching)
+
+> [!WARNING]
+> These instructions grant permission to act as the managed identity to the pipelines where it is
+> used. Moreover, vcpkg caches provide a mechanism where a compromised build may be able cause
+> other builds to use compromised assets from caches.
+>
+> You should not allow machines which run untrusted 3rd party build requests to write to caches
+> that are used for official builds.
+
+## Prerequisites
+
+::: zone pivot="shell-cmd"
+
+>[!NOTE]
+> These instructions are impractical with 'CMD', please choose 'PowerShell' or 'Bash' at the top
+> of the page.
+
+::: zone-end
+
+::: zone pivot="shell-powershell"
+
+* A terminal with [Azure PowerShell](/powershell/azure/install-azps-windows) installed
+* [vcpkg](../get_started/get-started.md#1---set-up-vcpkg)
+* An Azure Storage subscription and container where you wish binaries to be stored
+
+::: zone-end
+
+::: zone pivot="shell-bash"
+
+* A terminal with [Azure CLI](/cli/azure/install-azure-cli-windows) installed
+* [vcpkg](../get_started/get-started.md#1---set-up-vcpkg)
+* An Azure Storage subscription and container where you wish binaries to be stored
+
+::: zone-end
+
+## 1 - Create an Entra Managed Identity
+
+Skip this step if you already have configured a
+[Managed Identity](/entra/identity/managed-identities-azure-resources/overview) you wish to use.
+
+In the Azure Portal, go to the page for a Resource Group in which you wish for the managed identity
+to be created, and press "+ Create". In the search box, enter "User Assigned Managed Identity".
+Choose "User Assigned Managed Identity" from "Microsoft" and choose "Create ->
+User Assigned Managed Identity". Name the identity, press "Review and Create", and "Create". For
+purposes of this documentation, the identity was named `vcpkg-docs-identity`.
+
+## 2 - Set Up Workload Identity Federation with Azure DevOps
+
+In the Azure DevOps portal for the project in which you wish to run Pipelines, select
+"Project Settings" in the lower left corner. Then select "Service Connections" on the left.
+Then choose "New Service Connection" on the right. Choose the "Azure Resource Manager" radio button,
+and press Next. Select "Workload identity federation (manual)" and press next. Name the
+service connection and press next. For purposes of this documentation, the service connection was
+named `vcpkg-docs-identity-connection`. At this point, Azure DevOps should be showing an issuer
+and subject identifier.
+
+In another tab, to the Azure Portal navigate to the managed identity created in step 1. On the left

```

Comment:
I think there's something wrong with this sentence, but I'm not entirely sure what was intended here.

Reply:
I think it just needs to say "In another tab, go to the Azure Portal and navigate to..."